src/components/StorageOverview/TopConsumers/TopConsumers.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { LineChart, Row, Col } from 'patternfly-react';
+
+import { InlineLoading } from '../../Loading';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext/StorageOverviewContext';
+import { getTopConsumerVectorStats } from '../../../selectors';
+
+const TopConsumersBody = ({ stats }) => {
+  let results = 'No data available';
+
+  if (stats.length) {
+    const columnsConf = getTopConsumerVectorStats(stats);
+    const { columns, unit } = columnsConf;
+    const formatTime = x => {
```

Comment:
`new Date(x).toISOString().substring(11,16)` is  a bit simpler. Could you please put this into utils? And maybe rename it as `formatToShortISOTime` ?

Reply:
**toISOString** method will return the time in zero UTC time zone. Instead of that **toString** method can be used.